ddtrace/ext/__init__.py

```diff
@@ -17,22 +14,3 @@ class SpanTypes(Enum):
     TEST = "test"
     WEB = "web"
     WORKER = "worker"
-
-
-@removals.removed_class("AppTypes")
```

Comment:
should we just updated this use our 1.0 deprecation warning?

someone might still be using this ?

Reply:
I think an argument could be made that since we've had this removal warning for 16 months it should be expected for users that could be using this, but you're right that the safer option is to just update to have the 1.0 version warning. @Kyle-Verhoog WDYT?

Reply:
Yeah since it has been deprecated for so long I was thinking that we can do away with it before 1.0 to ease some of the load of 1.0. We'll have a ton of things that will be included in 1.0 so any chance we have to lessen that would be good IMO.

I don't feel strongly about this though and I would also feel comfortable leaving it for 1.0 from a user standpoint.

Reply:
this is similar to the constants, I definitely know there are people using these. I hope with the warning they have moved to `SpanTypes`, but this warning doesn't really give a good suggestion for alternative, just that it is being removed.

Reply:
Well also AppTypes simply don't do anything anymore and haven't for a long time. If someone is using them then it's dead code on their side as well. 

> but this warning doesn't really give a good suggestion for alternative, just that it is being removed.

https://github.com/DataDog/dd-trace-py/blob/a515c53ff872f6096d55985b3e09fcd53081fd1b/ddtrace/utils/__init__.py#L10-L17

Reply:
Fair enough, I'll leave the code in with an update to the removal version 1.0.0 😄

Reply:
Oh, I thought it was a generic message, but the message is good as-is